config/routes.rb

```diff
@@ -1,6 +1,15 @@
 Rails.application.routes.draw do
+  use_doorkeeper
   devise_for :users, controllers: { sessions: 'users/sessions', omniauth_callbacks: 'users/omniauth_callbacks' }
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      resources :profiles, only: [] do
```

Comment:
Does it work ok?
Sometimes it needs resources :profiles, only: [], defaults: :json